### DISCUSSION TITLE
`install-service.bat` and `uninstall-service.bat` missing?

### DISCUSSION THREAD

**Discussion by sitiom** [LizardByte/community#40] [Sunshine]
In https://github.com/SunshineStream/Sunshine/releases/tag/v0.14.0, the `install-service.bat` and `uninstall-service.bat` seems to have been removed. It would be great to add them back to add the `sunshinevc` service easily.

Additionally, the Windows installer should be able to set up the service for you automatically.

**Comment by ReenigneArcher** [comment]
Please create an issue
